src/sentry/workflow_engine/endpoints/serializers.py

```diff
@@ -126,11 +128,30 @@ def get_attrs(
             for group, serialized in zip(condition_groups, serialize(condition_groups, user=user))
         }
 
+        filtered_item_list = [item for item in item_list if item.type == ErrorGroupType.slug]
+        project_ids = [item.project_id for item in filtered_item_list]
+
+        project_options_list = list(
+            ProjectOption.objects.filter(
+                key__in=Detector.error_detector_project_options.values(), project__in=project_ids
+            )
+        )
+
+        configs: dict[int, dict[str, Any]] = defaultdict(
+            dict
+        )  # make the config for Error Detectors
+        for option in project_options_list:
+            configs[option.project_id][option.key] = option.value
+
         for item in item_list:
             attrs[item]["data_sources"] = ds_map.get(item.id)
             attrs[item]["condition_group"] = condition_group_map.get(
                 str(item.workflow_condition_group_id)
             )
+            if item.id in configs:
+                attrs[item]["config"] = configs[item.id]
+            else:
+                attrs[item]["config"] = item.config
```

Comment:
for all other detectors, will this just be empty?

Reply:
it's possible, but it will have whatever shape defined by `detector_config_schema` for the group type https://github.com/getsentry/sentry/blob/228d7279060c6198143ba3a4a203d99cd4b059a5/src/sentry/issues/grouptype.py#L274

Reply:
^ as an example. but for the error detector i'm overwriting it since i'm not migrating the project options rn